webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -64,6 +64,11 @@
             </template>
           </extension-registry-components>
         </v-card-text>
+        <v-card-text>
+          <extension-registry-components
+            name="ActivityComposerKudos"
```

Comment:
Please avoid using `Kudos` term in Social
```suggestion
            name="..."
```

Reply:
done!